lang/en/maps/groups.php

```diff
@@ -33,8 +33,9 @@
         'until' => 'Create up to :max groups to each map.',
     ],
     'placeholders'  => [
-        'name'      => 'Shops, Treasure, NPCs',
-        'position'  => 'Last',
+        'name'           => 'Shops, Treasure, NPCs',
+        'position'       => 'First',
+        'position_list'  => 'After ',
```

Comment:
Last tweak, have :name as a param, because in some language the name will be before the translation for "after" :)